frappe/model/db_query.py

```diff
@@ -35,7 +35,7 @@ def execute(self, fields=None, filters=None, or_filters=None,
 		join='left join', distinct=False, start=None, page_length=None, limit=None,
 		ignore_ifnull=False, save_user_settings=False, save_user_settings_fields=False,
 		update=None, add_total_row=None, user_settings=None, reference_doctype=None,
-		run=True, strict=True, pluck=None, ignore_ddl=False, parent_doctype=None) -> List:
+		run=True, strict=True, pluck=None, ignore_ddl=False, parent_doctype=None, get_all=False) -> List:
```

Comment:
This doesn't require an extra kwarg. It's get_all if `ignore_permissions` is set.